### ISSUE TITLE
Tree shaking doesn't work

### ISSUE THREAD

**Issue by IgorBabkin** [mobxjs/mobx-utils#237]
dependencies

```
  "sideEffect": false,
  "devDependencies": {
    "@types/webpack-env": "^1.15.1",
    "awesome-typescript-loader": "^5.2.1",
    "clean-webpack-plugin": "^3.0.0",
    "tslint": "^6.0.0",
    "tslint-react": "^4.0.0",
    "typescript": "^3.2.1",
    "webpack": "^4.41.6",
    "webpack-bundle-analyzer": "^3.6.0",
    "webpack-cli": "^3.3.11",
    "webpack-dev-server": "^3.10.3",
    "webpack-notifier": "^1.7.0"
  },
  "dependencies": {
    "mobx": "^5.15.4",
    "mobx-utils": "^5.5.3"
  }
```

Webpack config

```
const path = require('path');
const WebpackNotifierPlugin = require('webpack-notifier');
const CleanWebpackPlugin = require('clean-webpack-plugin').CleanWebpackPlugin;
const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;

module.exports = {
    mode: 'production',
    context: path.resolve(__dirname, './src'),
    entry: {
        app: './app',
    },
    output: {
        filename: '[name].[hash].js',
        path: path.resolve(__dirname, './dist'),
    },

    optimization: {
        splitChunks: {
            chunks: "all",
        },
        minimize: true,
    },

    resolve: {
        extensions: [".ts", ".json", ".js"],
        modules: [
            path.resolve(__dirname, './src'),
            'node_modules'
        ]
    },

    module: {
        rules: [
            {
                test: /\.tsx?$/,
                exclude: /node_modules/,
                loader: "awesome-typescript-loader"
            }
        ]
    },

    plugins: [
        new CleanWebpackPlugin(),
        new WebpackNotifierPlugin(),
        new BundleAnalyzerPlugin({
            analyzerMode: 'static',
            openAnalyzer: true,
        }),
    ],
};
```

tsconfig.json

```
{
  "compilerOptions": {
    "baseUrl": "./src",
    "module": "ES6",
    "moduleResolution": "node",
    "target": "ES5"
  },
  "exclude": [
    "node_modules"
  ]
}

```

Code 
```
import {NOOP} from "mobx-utils";

console.log(NOOP());
```

**Comment by IgorBabkin** [comment]
<img width="248" alt="Screenshot 2020-02-17 at 8 43 10 PM" src="https://user-images.githubusercontent.com/1428790/74682407-3dc64580-51c6-11ea-9d6c-ba9aa7068998.png">
<img width="1426" alt="Screenshot 2020-02-17 at 8 42 51 PM" src="https://user-images.githubusercontent.com/1428790/74682411-40289f80-51c6-11ea-86f4-f2a5e70172c0.png">

**Comment by IgorBabkin** [comment]
<img width="441" alt="Screenshot 2020-02-17 at 8 46 07 PM" src="https://user-images.githubusercontent.com/1428790/74682512-8bdb4900-51c6-11ea-83d3-93084ae627c6.png">

**Comment by IgorBabkin** [comment]
![image](https://user-images.githubusercontent.com/1428790/74843233-eb9f3480-532b-11ea-8ada-71762872ef16.png)

**Comment by mweststrate** [comment]
Should be fixed in 5.5.5
